scripts/hud/jump-stats.ts

```diff
@@ -57,38 +86,38 @@ class JumpStats {
 		this.setText();
 	}
 
-	static initializeBuffer(size) {
-		const buffer = Array.from({ length: size }).fill('\n');
+	initializeBuffer(size: number): string[] {
+		const buffer = Array.from({ length: size }).fill('\n') as string[];
 		buffer[buffer.length - 1] = '';
 		return buffer;
 	}
 
-	static addToBuffer(buffer, value) {
+	addToBuffer(buffer: string[], value: string | number) {
 		buffer[buffer.length - 1] += '\n';
-		buffer.push(value);
+		buffer.push(value.toString());
```

Comment:
I would rather add `.toFixed(0)` to `jumpCount` and `strafeCount` on lines 56, 69, and 74 and disallow `value` from being a `number`. I think `.toString()` on a `string` will be really cheap in v8 - this just feels extra to me. What do you think?

Reply:
Gotcha, that seems a lot better. I was struggling to understand the types in this class, this makes a lot a sense.